calico_containers/calico_ctl/node.py

```diff
@@ -84,14 +88,17 @@ def node(arguments):
     elif arguments.get("stop"):
         node_stop(arguments.get("--force"))
     else:
+        assert arguments.get("--detach") in ["true", "false"]
+        detach = arguments.get("--detach") == "true"
         node_start(ip=arguments.get("--ip"),
                    node_image=arguments['--node-image'],
                    log_dir=arguments.get("--log-dir"),
                    ip6=arguments.get("--ip6"),
-                   as_num=arguments.get("--as"))
+                   as_num=arguments.get("--as"),
+                   detach=detach)
 
 
-def node_start(node_image, log_dir, ip="", ip6="", as_num=None):
+def node_start(node_image, log_dir, ip="", ip6="", as_num=None, detach=True):
```

Comment:
probably shouldn't be defaulting any args, given how this is used.  Any objection?